napari/plugins/_tests/test_npe2.py

```diff
@@ -42,6 +42,18 @@ def test_read(mock_pm: 'TestPluginManager'):
     assert _npe2.read(["some.randomext"], stack=True) is None
     mock_pm.commands.get.assert_not_called()
 
+    mock_pm.commands.get.reset_mock()
+    assert (
+        _npe2.read(["some.randomext"], stack=True, plugin='not-npe2-plugin')
+        is None
+    )
+    mock_pm.commands.get.assert_not_called()
+
+    with pytest.raises(
```

Comment:
oooops, sorry @p-j-smith `min_req` tests are failing now - we should split out and skip this test if npe2 < 0.7:

```python
@pytest.mark.skipif(npe2.__version__ < '0.7.0', reason='Older versions of npe2 do not throw specific error.')
def test_read_with_plugin_failure(mock_pm: 'TestPluginManager'):
    with pytest.raises(
        ValueError, match=f"Plugin '{PLUGIN_NAME}' was selected"
    ):
        _npe2.read(["some.randomext"], stack=True, plugin=PLUGIN_NAME)
```

Reply:
ooo thanks for spotting that and providing the new test